src/components/HintIcon/HintIcon.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isNotUndefined } from 'ramda-adjunct';
+import { includes } from 'ramda';
+
+import { getColor } from '../../utils';
+import { Tooltip } from '../Tooltip';
+import { Icon } from '../Icon';
+import { HintIconProps } from './HintIcon.types';
+import { ColorTypes } from '../../theme';
+import { Color } from '../../theme/colors.types';
+
+const StyledIcon = styled(Icon).withConfig<{ color: Color }>({
+  shouldForwardProp: (property) => !includes(property, ['color']),
+})`
+  background: ${getColor('neutral.0')};
+  border: 1px solid ${getColor('neutral.0')};
+  border-radius: 100%;
+  color: ${({ color, theme }) =>
+    isNotUndefined(color) ? getColor(color, { theme }) : 'inherit'};
+  &:focus-visible,
+  &:hover {
+    color: ${({ theme }) => getColor('neutral.700', { theme })};
+  }
+`;
+
+const HintIcon: React.FC<HintIconProps> = ({
+  placement,
+  popup,
```

Comment:
Since we dictate the trigger for the HintTooltip it would be much better to pass tooltip content as classic `children`